webapp/src/js/enketo/widgets/mrdt.js

```diff
@@ -53,14 +53,23 @@ define( function( require, exports, module ) {
         }
 
         $el.on( 'click', '.btn.mrdt-verify', function() {
-            service.verify().then( function(image) {
+            service.verify().then( function(data) {
+                var image = data.image;
+                var timeTaken = data.timeTaken;
                 $( self.element )
                     .find( 'textarea' )
                     .val( image )
                     .trigger( 'change' );
                 $( self.element )
                     .find( '.mrdt-preview' )
                     .attr('src', 'data:image/png;base64, ' + image);
+                if (timeTaken) {
+                  $( self.element )
+                      .siblings( '.or-appearance-mrdt-time-taken' )
```

Comment:
Currently we are using a combination of order and the `appearance` column in xlsx to identify the input elements. For example, if the `mrdt-time-taken` field is first, it will be used as the image and the time-taken field. We should consolidate on one.